thirdparty/git_test.go

```diff
@@ -87,3 +87,43 @@ func TestParseGitUrl(t *testing.T) {
 	assert.Equal("evergreen-ci", owner)
 	assert.Equal("sample", repo)
 }
+
+func TestIsVersionMinimum(t *testing.T) {
+	tests := []struct {
+		version    string
+		minVersion string
+		expected   bool
+	}{
+		{"2.37.0", "2.380", true},
+		{"2.38.0", "2.380", true},
+		{"2.38.0", "2.38.1", true},
+		{"2.38.1", "2.38.0", false},
```

Comment:
I've been looking at these tests for a second, but I think something is backwards or unintuitive.

I'm comparing the first to the second, checking if the first meats the minimum version (which is the second). In this case (L100), 2.38.1 is past 2.38.0 by a patch version, but you expect false. It seems like all the test cases have the opposite of what I would expect, could you comment or switch some logic around if I'm interpreting it right?

Reply:
That is very true, it was flipped. I'm confused about how this seemed to work while I was testing on staging, maybe I moved things around after testing...? Thank you for catching that!

Reply:
Okay so after some more testing and reading, I figured out what happened here. "IsVersionMinimum" checked if it's the minimum or less, and then IsGitVersionMinimum returned  !IsVersionMinimum.  The naming was not great. I flipped both and made it more clear.